scipy/special/cephes/igam.c

```diff
@@ -164,9 +170,15 @@ double igamc(double a, double x)
 {
     double absxma_a;
 
-    if (x < 0 || a <= 0) {
+    if (x < 0 || a < 0) {
 	sf_error("gammaincc", SF_ERROR_DOMAIN, NULL);
 	return NPY_NAN;
+    } else if (a == 0) {
+        if ( x > 0 ) {
+		return 0;
+	} else {
+		return NPY_NAN;
```

Comment:
Looks like your spacing settings are off-we want to indent by 4 spaces.

Reply:
not sure how the tabs got in there, fixed